### PR TITLE
[Autocomplete] Translate "Add ..." text

### LINKED ISSUE #745: [Autocomplete] Translate "Add"
Seems that it is not possible to translate "Add "<item>". Please add that possibility :)

### PR DESCRIPTION
**Pull request by jmsche**
| Q             | A
| ------------- | ---
| Bug fix?      | no
| New feature?  | yes
| Issues        | Fix #745 
| License       | MIT

Hi, this PR fixes #745, allowing to translate the "Add Abc..." text when the create option is set to true :) 

Added the translated text as comment in most translation files.

image

### CODE REVIEW

**Comment by jmsche** [comment]
UX Website & Live/Twig components workflow failures are unrelated (due to Twig release from yesterday I guess).

Not sure why the Autocomplete workflow fails, the tests are working locally :/

File: src/Autocomplete/assets/src/controller.ts [hunk 1]
```diff
@@ -136,6 +145,9 @@ export default class extends Controller {
             no_results: () => {
                 return `<div class="no-results">${this.noResultsFoundTextValue}</div>`;
             },
+            option_create: (data: TomOption, escapeData: typeof escape_html): string => {
+                return `<div class="create">${this.createOptionTextValue} <strong>${escapeData(data.input)}</strong>&hellip;</div>`;
```
Comment (Kocal) [inline src/Autocomplete/assets/src/controller.ts]: It works for french and english (and surely more languages), but is there a lang when you expect the `data.input` to prepend `this.createOptionTextValue`? 

Something like `<div class="create"><strong>${escapeData(data.input)}</strong> ${this.createOptionTextValue} &hellip;</div>`?

Comment (jmsche) [reply src/Autocomplete/assets/src/controller.ts]: I can make some change, adding some `%placeholder%` in the translation and replace it with JS?

Comment (Kocal) [reply src/Autocomplete/assets/src/controller.ts]: I think that would be better yes

Comment (jmsche) [reply src/Autocomplete/assets/src/controller.ts]: Done :)

**Comment by smnandre** [comment]
@jmsche Did you check if passing option_create every time to the instance would not be interpreted by TomSelect as "create: true" ? 

I believe it worked like that at a time 

I would not want we add an unwanted option to every existing user :)

**Comment by jmsche** [comment]
@smnandre Just checked locally, adding autocomplete on an EnumType, create option is not offered so everything seems okay :)

**Review by smnandre — APPROVED** [review]
Could you update the CHANGELOG ? :)

**Comment by jmsche** [comment]
@smnandre Changelog entry added :)

**Comment by smnandre** [comment]
The Autocomplete test (PHP 8.3) are failing :|

**Review by smnandre — CHANGES_REQUESTED** [review]
Let's fix the tests and merge this :)

**Comment by jmsche** [comment]
@smnandre TBH I don't know why the `testItReturnsBasicResults` test is failing.
I've checked the code, and it shouldn't as there's only one fixture using the word "bar".

This test is passing on my local machine, and I guess if you run it again on CI it should pass :sweat_smile:

File: src/Autocomplete/CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,9 @@
 # CHANGELOG
 
+## 2.18.0
```
Comment (smnandre) [inline src/Autocomplete/CHANGELOG.md]:
```suggestion
## 2.20.0
```

**Review by smnandre — APPROVED** [review]

File: src/Autocomplete/translations/AutocompleteBundle.hu.php [hunk 3]
```diff
@@ -13,4 +13,5 @@
     'Loading more results...' => 'További találatok betöltése...',
     'No results found' => 'Nincs találat',
     'No more results' => 'Nincs több találat',
+    // 'Add %placeholder%...' => 'Add %placeholder%...',
```
Comment (1ed) [inline src/Autocomplete/translations/AutocompleteBundle.hu.php]:
```suggestion
    'Add %placeholder%...' => '"%placeholder%" hozzáadása',
```

**Review by Kocal — APPROVED** [review]

**Comment by smnandre** [comment]
Thanks @jmsche.

**Pull request merged**